velox/functions/prestosql/window/tests/NthValueTest.cpp

```diff
@@ -95,33 +116,39 @@ class MultiNthValueTest : public NthValueTest,
   MultiNthValueTest() : NthValueTest(GetParam()) {}
 };
 
-// Tests nth_value with data of a uniform distribution.
+// Tests nth_value, first_value, last_value functions with data of a uniform
```

Comment:
Maybe write "(nth|first|last)_value" functions in all the comments below to make them more concise.